errors/errors.go

```diff
@@ -173,17 +160,76 @@ func IsUserDoesNotExistError(err error) bool {
 	return isSpecificError(err, "UserDoesNotExist")
 }
 
-// isSpecificError checks whether the error represented by err wraps
-// an underlying MantaError with code errorCode.
-func isSpecificError(err error, errorCode string) bool {
-	tritonErrorInterface := errwrap.GetType(err.(error), &MantaError{})
-	if tritonErrorInterface == nil {
-		return false
-	}
+func IsBadRequest(err error) bool {
+	return isSpecificError(err, "BadRequest")
+}
+
```

Comment:
Can you make sure the error funcs in this file are either one space apart or have no spaces between them, not both? Just for the sake of consistency.